src/game/client/components/menus.cpp

```diff
@@ -971,7 +983,7 @@ CMenus::CListboxItem CMenus::UiDoListboxNextItem(CListBoxState* pState, const vo
 		{
 			pState->m_ListBoxDoneEvents = 1;
 
-			if(m_EnterPressed || (UI()->CheckActiveItem(pId) && Input()->MouseDoubleClick()))
+			if(m_EnterPressed || (UI()->LastActiveItem() == pId && Input()->MouseDoubleClick()))
```

Comment:
Don't do that. Could lead to a GUI lock.

Reply:
Ok, fixed.